app/src/test/java/org/simple/clinic/user/UserSessionTest.kt

```diff
@@ -836,6 +854,9 @@ class UserSessionTest {
     val preferencesEditor = mock<SharedPreferences.Editor>()
     whenever(preferencesEditor.clear()).thenReturn(preferencesEditor)
     whenever(sharedPrefs.edit()).thenReturn(preferencesEditor)
+    whenever(preferencesEditor.putString(eq("key"), any())) doReturn preferencesEditor
+    whenever(selectedCountryPreference.key()) doReturn "key"
+    whenever(sharedPrefs.getString(eq("key"), any())) doReturn ""
```

Comment:
Ufff... these tests 😅. Cascading changes across tests. Guess, we already have a pivotal ticket for refactoring `UserSession`?

Reply:
Yes, a lot of things will be broken out of `UserSession` once the breaking up of the DI scope happens. We can revisit how much `UserSession` needs to be refactored out after.